src/Workspaces/Core/Portable/Workspace/Solution/StateChecksums.cs

```diff
@@ -61,9 +79,10 @@ public static SolutionStateChecksums Deserialize(ObjectReader reader)
     {
         var checksum = Checksum.ReadFrom(reader);
         var result = new SolutionStateChecksums(
-            attributesChecksum: Checksum.ReadFrom(reader),
-            projectChecksums: ChecksumCollection.ReadFrom(reader),
-            analyzerReferenceChecksums: ChecksumCollection.ReadFrom(reader),
+            attributes: Checksum.ReadFrom(reader),
+            reader.ReadArray(ProjectId.ReadFrom),
```

Comment:
>reader.ReadArray(ProjectId.ReadFrom), [](http://example.com/codeflow?start=12&length=37)

nit: named param to match others would be nice

Reply:
done!